node/src/service.rs

```diff
@@ -474,11 +477,12 @@ pub fn new_full<
 									if futures::executor::block_on(
 										worker_handle.submit(seal.encode()),
 									) {
-										log::info!("🥇Successfully mined and submitted a new block via external miner");
+										let mining_time = mining_start_time.elapsed().as_secs();
```

Comment:
Is mining time purely for printing it out on the command line?

Reply:
yes